### ISSUE TITLE
Google search result summary is strange

### ISSUE THREAD

**Issue by armanbilge** [http4s/http4s#6943]
We set the appropriate meta tag here I believe, so I don't understand why Google isn't respecting it.
https://github.com/http4s/http4s/blob/cb97abaccd1d7253c6175ca7319b49f8a4ca25d4/index.html#L51

<img width="627" alt="Screen Shot 2023-01-25 at 3 01 43 PM" src="https://user-images.githubusercontent.com/3119428/214711433-12a548da-8681-4d79-97b8-0ce11cf2f292.png">

**Comment by aeons** [comment]
The docs say they might use the description meta tag: https://developers.google.com/search/docs/appearance/snippet

**Comment by armanbilge** [comment]
Thanks for that link!

> Google sometimes uses <meta> tag content to generate snippets, if we think they give users a more accurate description than can be taken directly from the page content.

So if I'm reading that correctly basically our meta tag is too shitty 😂

**Comment by katiemarqueling** [comment]
I agree with @armanbilge that description is too short. I checked, and it looks like Google is now rendering a new description. Would you like something different still? Id be happy to write something. 

![image](https://user-images.githubusercontent.com/33237218/220703015-258e70a6-6927-474b-b119-bb07c365bf01.png)

**Comment by armanbilge** [comment]
Oh, I guess they rotate that huh 😅 still that doesn't really explain what http4s is.

So @katiemarqueling if you want to write something, sure, that would be great! I still need to figure out where exactly it should go, maybe in here.
https://github.com/http4s/http4s/blob/4bcefdd689cce3110b095276ba11187827fa3a87/project/Http4sSitePlugin.scala#L60-L61
